recipe_scrapers/_schemaorg.py

```diff
@@ -186,8 +186,8 @@ def image(self):
             image = image.get("url")
 
         if "http://" not in image and "https://" not in image:
-            # some sites give image path relative to the domain
-            # in cases like this handle image url with class methods or og link
+            # Some sites give image path relative to the domain.
+            # In cases like this handle image url with class methods or og link
```

Comment:
```suggestion
            # Some sites use relative image paths;
            # prefer generic image retrieval code in those cases.
```

...I'm not completely sure that's what we want to do, but looking at the code, it's what it does, so it's good to document.